jenkins/ompi/Dockerfile.centos

```diff
@@ -40,7 +40,18 @@ ENV MOFED_SITE_PLACE MLNX_OFED-${OMPI_CI_MOFED_VERSION}
 ENV MOFED_IMAGE MLNX_OFED_LINUX-${OMPI_CI_MOFED_VERSION}-rhel7.6-x86_64.tgz
 RUN wget --no-verbose http://content.mellanox.com/ofed/${MOFED_SITE_PLACE}/${MOFED_IMAGE} && \
     tar -xzvf ${MOFED_IMAGE} && \
-    ${MOFED_DIR}/mlnxofedinstall --user-space-only --without-fw-update --all -q --skip-distro-check && \
+    ${MOFED_DIR}/mlnxofedinstall \
+        --user-space-only \
+        --without-fw-update \
+        --all \
+        -q \
+        --skip-distro-check \
+        --without-ucx \
+        --without-hcoll \
+        --without-mxm \
+        --without-openmpi \
+        --without-sharp \
+        && \
```

Comment:
Why do you need this trailing `&& \`?

Reply:
For better readability in case there are a lot of arguments.